docs/03-tutorials/00-eventing/evnt-01-send-legacy-events.md

```diff
@@ -0,0 +1,221 @@
+      serverless.kyma-project.io/replicas-preset: S
+    name: lastorder
+    namespace: default
+  spec:
+    deps: '{ "dependencies": {}}'
+    maxReplicas: 1
+    minReplicas: 1
+    source: |
+      module.exports = {
+        main: async function (event, context) {
+          console.log("Received event:", event.data);
+          return; 
+        } 
+      }
+EOF
+```
+
+If the resources were created successfully, the command returns this message:
+
+```bash
+function.serverless.kyma-project.io/lastorder created
+```
+
+To check the Function status, run:
+
+```bash
+kubectl get functions -n default lastorder
+```
+
+> **NOTE:** You might need to wait a few seconds for the Function to be ready.
+
+  </details>
+</div>
+
+## Create a Subscription
+
+Next, to subscribe to events, we need a [Subscription](../../05-technical-reference/00-custom-resources/evnt-01-subscription.md) custom resource. We're going to subscribe to events of the type `order.received.v1`.
+All the published events of this type are then forwarded to an HTTP endpoint called `Sink`. You can define this endpoint in the Subscription's spec.
+
+<div tabs name="Create a Subscription" group="trigger-workload">
+  <details open>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+1. In Kyma Dashboard, go to the view of your Function `lastorder`.
+2. Go to **Configuration** > **Create Subscription+**.
+3. Provide the following parameters:
+   - **Subscription name**: `lastorder-sub`
+   - **Application name**: `myapp`
+   - **Event name**: `order.received`
+   - **Event version**: `v1`
+
+   - **Event type** is generated automatically. For this example, it's `sap.kyma.custom.myapp.order.received.v1`.
+
+4. Click **Create**.
+5. Wait a few seconds for the Subscription to have status `READY`.
+
+  </details>
+  <details>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+```bash
+cat <<EOF | kubectl apply -f -
+   apiVersion: eventing.kyma-project.io/v1alpha1
+   kind: Subscription
+   metadata:
+     name: lastorder-sub
+     namespace: default
+   spec:
+     filter:
+       filters:
+       - eventSource:
+           property: source
+           type: exact
+           value: ""
+         eventType:
+           property: type
+           type: exact
+           value: sap.kyma.custom.myapp.order.received.v1
+     sink: http://lastorder.default.svc.cluster.local
+EOF
+```
+
+To check that the Subscription was created and is ready, run:
+```bash
+kubectl get subscriptions lastorder-sub -o=jsonpath="{.status.ready}"
+```
+
+The operation was successful if the returned status says `true`.
+
+  </details>
+</div>
+
+## Publish a legacy event to trigger the workload
+
+We created the `lastorder` Function, and subscribed to the `order.received.v1` events by creating a Subscription CR. Now it's time to send an event and trigger the Function. In this example, we'll port-forward the Kyma Eventing Service to localhost.
```

Comment:
```suggestion
We created the `lastorder` Function, and subscribed to the `order.received.v1` events by creating a Subscription CR. Now it's time to send an event and trigger the Function. In this example, we'll port-forward the Kyma Eventing Publishing Service to localhost.
```
Should we call this `Kyma Eventing Publishing Service`? Our component has multiple services and imho we should encode part of the functionality in the Terms that we use. I hope this would make it easier to understand for our users.
Once we decided a name we should put it in a glossar in our documentation ...

Reply:
Then let's use the actual `Eventing Publisher Proxy Service`

Reply:
👍